grails-app/domain/org/broadinstitute/orsp/StorageDocument.groovy

```diff
@@ -31,6 +32,10 @@ class StorageDocument {
         status nullable: false
     }
 
+    static mapping = {
+        version column: '`version`'
```

Comment:
Can we use a different name here, something like `document_version`? Grails/GORM auto-adds a `version` column to all mapped entities and I don't want to get confused between what it means here as compared to other entities.